packages/client/src/containers/Dashboard/Dashboard.css

```
@@ -139,3 +138,99 @@ body {
   align-items: flex-start;
   flex: 1 0 0;
 }
+
+.team-members-container {
+  display: flex;
+  flex-direction: column;
+  flex: 1 0 0;
+  margin-left: 30px;
+  margin-right: 30px;
+}
+
+.teamMembers-heading {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start; /* Align the text to the left */
+  margin-bottom: 20px; /* Add margin for spacing */
+}
+
+.team-members-container ul {
+  list-style-type: none; /* Remove default list styles */
+  padding: 0; /* Remove default padding */
+  background: var(--primary-background-color);
+  padding: 20px 20px 20px 20px;
```

There is a short hand for this:
```suggestion
  padding: 20px;
```